tests/integration/profiling/rule_based_profilers/test_profiler_user_workflows.py

```diff
@@ -1440,3 +1440,73 @@ def test_quentin_expect_column_quantile_values_to_be_between_auto_yes_default_pr
         profiler_config=custom_profiler_config,
     )
     assert result.success
+
+
+@probabilistic_test
+@pytest.mark.skipif(
+    version.parse(np.version.version) < version.parse("1.21.0"),
+    reason="requires numpy version 1.21.0 or newer",
+)
+@freeze_time("09/26/2019 13:42:41")
+def test_quentin_expect_column_values_to_be_in_set_auto_yes_default_profiler_config_yes_custom_profiler_config_no(
+    quentin_columnar_table_multi_batch_data_context,
+):
+    context: DataContext = quentin_columnar_table_multi_batch_data_context
+
+    result: ExpectationValidationResult
+
+    custom_profiler_config: RuleBasedProfilerConfig
+
+    expectation_suite_name: str = f"tmp.profiler_suite_{str(uuid.uuid4())[:8]}"
+    try:
+        # noinspection PyUnusedLocal
+        suite = context.get_expectation_suite(
+            expectation_suite_name=expectation_suite_name
+        )
+    except ge_exceptions.DataContextError:
+        suite = context.create_expectation_suite(
+            expectation_suite_name=expectation_suite_name
+        )
+        print(f'Created ExpectationSuite "{suite.expectation_suite_name}".')
```

Comment:
We should know here whether the suite exists so you can probably get rid of the try/except

Reply:
@anthonyburdi this is the "safe" pattern from elsewhere (not thinking there is any harm in keeping it like this).